cdap-ui/app/cdap/components/Wizard/index.js

```diff
@@ -308,5 +307,6 @@ Wizard.propTypes = {
     replaceReducer: PropTypes.func
   }).isRequired,
   onSubmit: PropTypes.func.isRequired,
-  onClose: PropTypes.func.isRequired
+  onClose: PropTypes.func.isRequired,
+  inputIsEmpty: PropTypes.bool
```

Comment:
This property might not be evident to disable the submit button. Why not have it as disabled. Could be more straight forward